EasyPost.Net/Client.cs

```diff
@@ -99,24 +109,24 @@ internal T Execute<T>(Request request, string rootElement = null) where T : new(
             }
 
             Dictionary<string, Dictionary<string, object>> body;
-            List<Error> errors;
+            Error error;
```

Comment:
What changed so this isn't a list anymore?

Reply:
Rolled back in fed1cecdc0406fa2d069b1b422f9d238927d4265, will address in a future PR